keyboards/1upkeyboards/1up60rgb/keymaps/mdyevimnav/keymap.c

```diff
@@ -0,0 +1,48 @@
+#include "1up60rgb.h"
```

Comment:
You can actually just use this

```suggestion
#include QMK_KEYBOARD_H
```

Reply:
Thx, fixed!